app/models/kudo.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Kudo < ApplicationRecord
+  validates :title, presence: true, length: { minimum: 2 }
+  validates :content, presence: true, length: { minimum: 2 }
+  validates :giver_id, presence: true
+  validates :receiver_id, presence: true
```

Comment:
you don't need this - the `belongs_to` is already taking care of that by default

If you do not add  `optional: true` option to `belongs_to` it is expecting Kudo to have that association